systemtest/cmd/apmbench/main.go

```diff
@@ -49,6 +51,12 @@ func Benchmark1000Transactions(b *testing.B, l *rate.Limiter) {
 }
 
 func BenchmarkOTLPTraces(b *testing.B, l *rate.Limiter) {
+	switch strings.ToLower(os.Getenv("OTLPTRACES_TEST")) {
```

Comment:
Seems a bit more suitable than suffixing `_TEST`?
```suggestion
	switch strings.ToLower(os.Getenv("RUN_OTLPTRACES")) {
```